### ISSUE TITLE
Automatically build MDANSE into packages installable using pip

### ISSUE THREAD

**Issue by MBartkowiakSTFC** [ISISNeutronMuon/MDANSE#207]
Now that we have switched to python 3 and do not need custom builds of any libraries, it is possible to install all the dependencies just by `pip install`. Therefore, we should be able to make MDANSE itself into a package accessed via pip. This would simplify the entire installation process **a lot**.

**Comment by MBartkowiakSTFC** [comment]
Step 1 of the procedure is complete:
We have a workflow in GitHub Actions which build wheels for a few platforms.

I just tried it out on Windows using Python 3.11. I had to download the wheel manually from here
https://github.com/ISISNeutronMuon/MDANSE/actions/runs/5012601660
install it manually using 
`pip install MDANSE-2.0.0-cp311-cp311-win_amd64.whl`
and **also manually** install the missing dependencies.

In the end I could start the existing part of the GUI by typing
`python mdanse_gui`
in the python virtual environment Scripts folder.

@eurydice76 It is still a long way to go before we can get MDANSE from pypi, but it is a start.
